src/deploy-cromwell-on-azure/Utility.cs

```diff
@@ -86,6 +86,12 @@ public ConfigReplaceRegExItemEvaluator(string match, MatchEvaluator replacement,
         public static string DictionaryToDelimitedText(Dictionary<string, string> dictionary, string fieldDelimiter = "=", string rowDelimiter = "\n")
             => string.Join(rowDelimiter, dictionary.Select(kv => $"{kv.Key}{fieldDelimiter}{kv.Value}"));
 
+        public static Stream GetFileStream(params string[] pathComponentsRelativeToAppBase)
```

Comment:
How is this different than `GetBinaryFileContent()`?

Reply:
None, fixed.